velox/docs/monthly-updates/may-2023.rst

```diff
@@ -0,0 +1,76 @@
+********************
+May 2023 Update
+********************
+
+Documentation
+=============
+
+ * Add `blog post on efficient complex types <https://velox-lib.io/blog/simple-functions-2>`_.
+
+
+Core Library
+============
+
+ * Add support to implement DECIMAL types as logical type.
```

Comment:
`DECIMAL types` -> `DECIMAL type`
I know the PR title says `types`. That should have been `type` as well.